dspace-oai/src/main/java/org/dspace/xoai/app/XOAI.java

```diff
@@ -300,12 +304,20 @@ private int index(Iterator<Item> iterator)
                     log.error(ex.getMessage(), ex);
                 }
                 i++;
-                if (i % 100 == 0) {
+                if (i % 1000 == 0) {
```

Comment:
Note that we get the message twice when oai.import.batch.size is defaulted.  Cosmetic.  But do we really need this?

Reply:
yes good point - i was going to leave this as is to be a straight 6.x port, but might as well tidy up here then do another small change to 6.x to match it
so now, that test is `if(i % 1000 == 0 && batchSize != 1000)`

Reply:
Sounds well, thanks!